sdk/resourcemanager/agrifood/armagrifood/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 0.8.1 (2023-04-14)
+### Bug Fixes
+
+- Fix serialization bug of empty value of `any` type.
```

Comment:
I am not aware of this fix.  Is it a bug fixing for the CodeGen and do we have an issue for this?

Reply:
Will this fix cause breaking change to the SDK?  @tadelesh    @raych1